scheduler/pkg/agent/k8s/secrets_test.go

```diff
@@ -67,7 +67,7 @@ parameters:
 		t.Run(test.name, func(t *testing.T) {
 			fakeClientset := fake.NewSimpleClientset(test.secret)
 			s := NewSecretsHandler(fakeClientset, test.secret.Namespace)
-			data, err := s.GetSecretConfig(test.secretName)
+			data, err := s.GetSecretConfig(test.secretName, 1)
```

Comment:
Isn't this converted into a "1 nanosecond" Duration by default? Even for a mock client, perhaps it would be better to set it to `1 * Millisecond` to avoid any flaky tests.